nixos/modules/services/misc/nixos-manual.nix

```diff
@@ -58,6 +58,15 @@ let
       exec "$browser" ${entry}
     '';
 
+  desktopItem = pkgs.makeDesktopItem {
+    name = "nixos-manual";
+    desktopName = "NixOS Manual";
+    genericName = "View NixOS documentation in a web browser";
+    # TODO: find a better icon (Nix logo + help overlay?)
+    icon = "system-help";
+    exec = "xdg-open ${entry}";
```

Comment:
re URLs, I am pretty sure that the URL field only applies to desktop files with `Type=Link`.  And for some reason the gnome and KDE app menus refuse to work with those; they require `Type=Application`, which in turn requires an `exec` line.  I'll switch to having it exec `${help}` though, that's a good idea.